vendor/github.com/containerd/containerd/cmd/containerd-shim/main_unix.go

```diff
@@ -62,7 +62,7 @@ var (
 	containerdBinaryArgv0Flag string
 )
 
-func init() {
+func Main() {
```

Comment:
Is this change related?

Reply:
this was already PR'd into balena-containerd here: https://github.com/balena-os/balena-containerd/pull/5

this commit updates the vendored copy: https://github.com/balena-os/balena-engine/pull/168/commits/9898babd1a7906365003dd940fa784a0c5a77cdb